### ISSUE TITLE
Create test cases for each component of the bot

### ISSUE THREAD

**Issue by joeldesante** [joeldesante/Harvey#18]
Test cases for each component of the bot need to be made.

**Comment by willuhmjs** [comment]
Creating test cases for discord.js is [notoriously annoying](https://github.com/discordjs/discord.js/issues/6798). However, I recommend using [mock objects](https://en.wikipedia.org/wiki/Mock_object) if you want to get the job done.

**Comment by joeldesante** [comment]
> Creating test cases for discord.js is [notoriously annoying](https://github.com/discordjs/discord.js/issues/6798). However, I recommend using [mock objects](https://en.wikipedia.org/wiki/Mock_object) if you want to get the job done.

Good idea. And yes, I think I tried to do this a while ago and gave up when I realized how annoying it can be. WIth that in mind, we still might be able to test individual units of the bot by directly calling the functions. And we can probably just assume that DiscordJS will work given that they probably test their releases on their end anyway.

**Comment by joeldesante** [comment]
Bumping this issue because I'm finding it challenging to test all of our bots commands and features manually.
